adobe/spectrum-web-components
# refactor: leverage Resize Controller over slotchange in item change trigger

## Description
Manage the Tab and Top Nav Item elements, their presence, and their size within their parent element with a shared Resize Controller. This will be more resilient to non-DOM effecting changes to size/content than `slotchange` listening.

## How has this been tested?
    1. Go [here](https://nav-tabs-item--spectrum-web-components.netlify.app/storybook/?path=/story/tabs--default)
    2. Add items, remove items, change the content of items, etc.
    3. Confirm that the Selection Indicator stay in/moves to its expected location on each change.
    1. Go [here](https://nav-tabs-item--spectrum-web-components.netlify.app/storybook/iframe.html?id=top-nav--selected&viewMode=story&args=#page-2)
    2. Add items, remove items, change the content of items, etc.
    3. Confirm that the Selection Indicator stay in/moves to its expected location on each change.

## Types of changes

## Checklist
